pkg/sync/filepath_sync.go

```diff
@@ -28,7 +34,37 @@ func (fs *FilePathSync) Fetch(_ context.Context) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return string(rawFile), nil
+
+	var r []byte
+	if fs.FileType == "yaml" {
```

Comment:
Should think about supporting. `.yml`

Reply:
Added support for `yml`. Thanks for pointing it out.